wagtail/admin/ui/side_panels.py

```diff
@@ -350,8 +354,23 @@ def __init__(self, object, request, *, preview_url):
         super().__init__(object, request)
         self.preview_url = preview_url
 
+    @property
+    def auto_update_interval(self):
+        if hasattr(settings, "WAGTAIL_AUTO_UPDATE_PREVIEW"):
+            warnings.warn(
+                "`WAGTAIL_AUTO_UPDATE_PREVIEW` is deprecated. "
+                "Set `WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL = 0` to disable "
+                "auto-update for previews.",
+                RemovedInWagtail70Warning,
+            )
+            if not settings.WAGTAIL_AUTO_UPDATE_PREVIEW:
+                return 0
+
+        return getattr(settings, "WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL", 500)
```

Comment:
Consolidating these two settings to follow the recent practice for `WAGTAIL_EDITING_SESSION_PING_INTERVAL`.

`WAGTAIL_AUTO_UPDATE_PREVIEW` has been around since before the preview panel was introduced, hence why we still used it when `WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL` was introduced.